symphony/bdk/core/config/model/bdk_client_config.py

```diff
@@ -0,0 +1,42 @@
+from symphony.bdk.core.config.model.bdk_server_config import BdkServerConfig
+
+
+class BdkClientConfig(BdkServerConfig):
+
+    def __init__(self, parent_config, config):
+        self.parent_config = parent_config
+        if config is not None:
+            self._scheme = config.get("scheme") if "scheme" in config else None
```

Comment:
https://docs.python.org/3/library/stdtypes.html#dict.get : get("key") on a dict defaults to `None` id key is not present.